use-timescale/data-tiering/about-data-tiering.md

```diff
@@ -82,14 +69,24 @@ ime zone))
 
 `EXPLAIN` illustrates which chunks are being pulled in from the object storage tier:
 
-1. Fetch data from chunks 42, 43, and 44 from the object storage tier. 
-1. Prune row groups and limit the fetch to a subset of the offsets in the
-Parquet object that potentially match the query filter. Only fetch the data
-for `device_uuid`, `sensor_id`, and `observed_at` as the query needs only these 3 columns.
+1. Fetch data from chunks 42, 43, and 44 from the object storage tier.
+1. Skip row groups and limit the fetch to a subset of the offsets in the
+   Parquet object that potentially match the query filter. Only fetch the data
+   for `device_uuid`, `sensor_id`, and `observed_at` as the query needs only these 3 columns.
+
+The object storage tier is more than an archiving solution. It is also:
+
+- **Cost-effective:** store high volumes of data at a lower cost. You pay only for what you store, with no extra cost for queries.
+- **Scalable:** scale past the restrictions of even the enhanced high-performance storage tier.
+- **Online:** your data is always there and can be [queried when needed][querying-tiered-data].
+
+By default, tiered data is not included when querying from a $CLOUD_LONG. However, you can access tiered data by [enabling tiered reads][querying-tiered-data] for a query, a session, or even for all sessions. After you enable tiered reads, when you run regular SQL queries, a behind-the-scenes process transparently pulls data from wherever it's located: the standard high-performance storage tier, the object storage tier, or both.  You can `JOIN` against tiered data, build views, and even define continuous aggregates on it. In fact, because the implementation of continuous aggregates also uses hypertables, they can be tiered to low-cost storage as well.
```

Comment:
```suggestion
By default, tiered data is not included when you query from a $SERVICE_LONG. To access tiered data, you [enable tiered reads][querying-tiered-data] for a query, a session, or even for all sessions. After you enable tiered reads, when you run regular SQL queries, a behind-the-scenes process transparently pulls data from wherever it's located: the standard high-performance storage tier, the object storage tier, or both.  You can `JOIN` against tiered data, build views, and even define continuous aggregates on it. In fact, because the implementation of continuous aggregates also uses hypertables, they can be tiered to low-cost storage as well.
```